Automattic/wp-calypso
# Performance Profiler: update recommendations section styling

Closes https://github.com/Automattic/dotcom-forge/issues/9115
Closes https://github.com/Automattic/dotcom-forge/issues/9114

## Proposed Changes

* update thumbsup/down icons
* Preserve the Did you know section column whitespace even when it’s not rendered

## Why are these changes being made?


pdt2If-1Jh-p2

## Testing Instructions



* Go to `/speed-test` and start a new test, or go to a results page eg. `/speed-test-tool?url=https%3A%2F%2Fwordpress.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NzA1N30.7KXaBL1qcm0y0GqvM8ymJ0pbST9ls6P09_hri8EaKSI`
* Check the styling of the recommendations section

| Section | Before | After |
|--------|--------|--------|
| Thumbsup/down icons | <img width="394" alt="image" src="https://github.com/user-attachments/assets/a6ebff4c-35bf-4d22-a32a-f36f2d664372"> | <img width="369" alt="image" src="https://github.com/user-attachments/assets/53dc8339-9c52-4296-b774-0c55263822d8"> |
| Did you know section column | <img width="1088" alt="image" src="https://github.com/user-attachments/assets/d6b3ca27-0495-4225-8ebc-9d67cbed2651"> | <img width="1068" alt="image" src="https://github.com/user-attachments/assets/679edcf9-1378-4a97-aa78-a0f541bee273"> |
| Cell | Cell | Cell |
| Cell | Cell | Cell |
| Cell | Cell | Cell |
